docs/user-guide/src/bmo/ironic_installation.md

```diff
@@ -0,0 +1,187 @@
+there is a high chance that you need to install Virtual Baseboard Management
+Controller (VBMC), which is virtual implementation of Baseboard Management
+Controller (BMC). For More info on VBMC check official [documentation](https://docs.openstack.org/virtualbmc/latest/).
+
+To run VMBC container:
+```bash
+ sudo docker run -d --net host \
+    --name vbmc \
+     -v "SOME_LOCAL_PATH":/root/.vbmc \
+     -v "/root/.ssh":/root/ssh \
+     quay.io/metal3-io/vbmc:latest
+```
+
+Our VMBC container image is available at [quay.io/metal3-io/vbmc](https://quay.io/repository/metal3-io/vbmc?tag=latest&tab=tags)
+
+### sushy-tools
+
+Sushy-tools is the same functionality as VBMC tool, except that it relies on [Redfish](https://www.dmtf.org/standards/redfish)
+protocol instead of IPMI. For more info, check the official [documentation](https://docs.openstack.org/sushy-tools/latest/index.html).
+
+Susy-tool accepts some configuration parameters, which
+we pass in the form of a file. Here is an example of `conf.py`
+configuration file. To see full list of supported parameters,
+check the sushy-tools [documentation](https://docs.openstack.org/sushy-tools/latest/admin/index.html).
+
+```python
+# The libvirt URI to use. This option enables libvirt drivers.
+# qemu+ssh://root@<remote-libvirt-server>/system?keyfile=<path-to-key>
+SUSHY_EMULATOR_LIBVIRT_URI = "qemu+ssh://root@192.168.111.1/system?&keyfile=/root/ssh/id_rsa_virt_power&no_verify=1&no_tty=1"
+SUSHY_EMULATOR_IGNORE_BOOT_DEVICE = False
+SUSHY_EMULATOR_VMEDIA_VERIFY_SSL = False
+```
+
+We will mount `conf.py` into the container's `/root/sushy/` path.
+
+```bash
+ sudo docker run -d --net host \
+    --name sushy-tools \
+     -v "SOME_PATH/conf.py":/root/sushy/conf.py \
+     -v "/root/.ssh":/root/ssh \
+     quay.io/metal3-io/sushy-tools:latest
+```
+
+Our sushy-tools container image is available at  [quay.io/metal3-io/sushy-tools](https://quay.io/repository/metal3-io/sushy-tools?tag=latest&tab=tags)
+
+### httpd
+
+We use this container to run an Apache web server for hosting node images. Below is the list of images (from `$IMAGES_PATH`) that we mount into the httpd container's `/shared` path.
+
+1. Ironic Python Images (IPA) via http for PXE booting
+    - ironic-python-agent.kernel
+    - ironic-python-agent.initramfs
+
+2. Target node image in qcow2 format
+    - bionic-server-cloudimg-amd64-disk.img
+
+We are not directly using Apache container image here, but instead [quay.io/metal3-io/ironic](https://quay.io/repository/metal3-io/ironic?tag=latest&tab=tags) which has `runhttpd` entry point to start off the httpd.
+
+`PROVISIONING_INTERFACE` - interface to use for ironic, dnsmasq (dhcpd) and httpd.
+
+```bash
+sudo docker run -d --net host --privileged \
+    --name httpd-infra \
+    -v "IMAGES_PATH":/shared \
+    --entrypoint /bin/runhttpd \
+    --env "PROVISIONING_INTERFACE=ironicendpoint" \
+    quay.io/metal3-io/ironic:latest
+```
+
+## Ironic in-cluster installation
```

Comment:
Okay, this is the recommended installation path. I'd put it first and highlight that this is the way we test and support.

Reply:
by first you mean at the top?